packages/components/psammead-amp-geo/src/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+const ampGeoData = {
+  ISOCountryGroups: {
+    eea: [
```

Comment:
Missing:
```
wf  Wallis & Futuna Islands
va  Vatican City
tf  French Southern Territories
sj  Svalbard & Jan Mayen Islands
pf  French Polynesia
nc  New Caledonia
ez  Eurozone
ea  Ceuta & Melilla
ax  Aland Islands
```

Reply:
Added all except for `ez` since although it's in the Google list of codes, I don't believe we'll want to add it in ours.